### DISCUSSION TITLE
Add copy button for OS and Chatterino version

### DISCUSSION THREAD

**Discussion by x9136** [Chatterino/chatterino2#4952] [Ideas] [answered]
Since it's worth specifying the OS and Chatterino version to create issues, it would be a nice little thing to copy that information in 1 click.

**Comment by pajlada** [comment]
You can already select the text and ctrl+c or right-click copy, that should be intuitive enough

**Reply by x9136** [reply]
I didn't say anything was unclear.

I said there is an opportunity to add a bit of convenience to everyone who will write to you about issues or tell you the OS and Chatterino version.

Instead of having to highlight everything and then hit a keyboard shortcut, there's an opportunity to do a nice little thing – a "Copy" button.

**Reply by pajlada** [reply, marked as answer]
I didn't imply you said anything was unclear, I was just explaining how you can do the same thing for anyone looking at the discussion in the future.
We will not add a copy button

**Reply by Nerixyz** [reply]
> We will not add a copy button

Why not?

It's not much effort to add a copy button, and I agree that it's easier to use a copy button than selecting the text, which, for new users, doesn't indicate the option to select the text unless you try it, hence the existence of this discussion.

**Reply by pajlada** [reply]
> Why not?

I don't think it's necessary to make a 2-step process a 1-step process - it's extra code & an extra element that has to be laid out in the dialog.

If we identify a problem where people don't know how to include the version text (which I don't think has been an issue so far), we can improve the issue template
